api/tests/unit/application/campaignReports/campaign-report-controller_test.js

```diff
@@ -0,0 +1,29 @@
+const { sinon, expect } = require('../../../test-helper');
+const campaignReportController = require('../../../../lib/application/campaignReports/campaign-report-controller');
+const campaignReportSerializer = require('../../../../lib/infrastructure/serializers/jsonapi/campaign-report-serializer');
+const usecases = require('../../../../lib/domain/usecases');
+
+describe('Unit | Controller | campaign-report-controller', () => {
+  describe('#get ', () => {
+
+    const campaignId = 1;
+
+    beforeEach(() => {
+      sinon.stub(usecases, 'getCampaignReport');
+      sinon.stub(campaignReportSerializer, 'serialize');
+    });
+
+    it('should returns ok', async () => {
```

Comment:
Pour avoir un nom de test plus explicite : 'should return ok when the campaign exists' ?

Et que se passe-t-il quand la campagne n'existe pas ?